syntax = "proto3";

option java_multiple_files = true;
option java_package = "io.grpc.examples.hello_agrpc";
option java_outer_classname = "HelloAgrpcProto";
option objc_class_prefix = "HA";

package hello_agrpc;

service Greeter {
  rpc SayHello(HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}
